### PR TITLE
quick create schedule

### PR DESCRIPTION
**Pull request by PhamVanTung**

### CODE REVIEW

File: app/assets/javascripts/calendar/custom.js [hunk 1]
```diff
@@ -215,6 +222,8 @@ $(document).ready(function() {
         });
         $(".datepicker").click(function(){
           $(".popover").hide();
+        element.click(function(){
```
Comment (ducnt-0269) [inline app/assets/javascripts/calendar/custom.js:226]: indent

File: app/views/shared/_event_short.html.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    <%= f.text_field :start_time %>
+    <%= t("pages.form.to") %>
+    <%= f.text_field :finish_time %>
+  </div>
+  <br/><br/>
+  <%= f.label t("schedules.form.event_name") %>
+  <%= f.text_field :title, placeholder: t("schedules.form.event_title"), class: "form-control", id: "title-schedule" %>
+  <br/>
+  <%= f.submit t("schedules.form.create_event"), class: "btn btn-primary submit-button" %>&nbsp;&nbsp;
+  <%= link_to "Edit event", new_schedule_path, title: :title, start_time: :start_time, finish_time: :finish_time %>
```
Comment (ducnt-0269) [inline app/views/shared/_event_short.html.erb]: I18n

File: app/views/pages/home.html.erb [hunk 3]
```diff
@@ -42,10 +42,8 @@
...
 
-<% content_for :modal_form do %>
-  <%= render "shared/modal_form" if current_user %>
-<% end %>
-
 <div class="form-popover hidden"><%= render "popover" %></div>
 
 <div id="time-table" class="hidden"><%= render "time_table" %></div>
+
+<div id="popup" style="display: none;"><%= render "shared/event_short" %></div>
```
Comment (NguyenTanDuc) [inline app/views/pages/home.html.erb]: css already have style display: none

Comment (ducnt-0269) [inline app/views/pages/home.html.erb]: css có style display: none rồi mà ông

File: app/assets/javascripts/calendar/custom.js [hunk 4]
```diff
@@ -321,3 +322,14 @@ $(document).ready(function() {
     MyCalendar.fullCalendar("rerenderEvents");
   }, 900000);
 });
+
+function formatDate(date) {
+  var date_of_month = date.getDate();
+  var month = date.getMonth()+1;
```
Comment (ducnt-0269) [inline app/assets/javascripts/calendar/custom.js]: space +

File: app/assets/javascripts/calendar/custom.js [hunk 5]
```diff
@@ -97,30 +102,32 @@ $(document).ready(function() {
 
       if((view.type != "month") && (start.date() == date.getDate())) {
         if((start.hour() > date.getHours()) || ((start.hour() == date.getHours()) && (start.minute() >= date.getMinutes()))) {
-          $("#modal-form").modal("show");
+          quick_create_schedule.css({
+            display: "block",
+            top:event.pageY - 280,
```
Comment (ducnt-0269) [inline app/assets/javascripts/calendar/custom.js]: space top:

File: app/assets/javascripts/calendar/custom.js [hunk 6]
```diff
@@ -321,3 +322,14 @@ $(document).ready(function() {
...
   }, 900000);
 });
+
+function formatDate(date) {
+  var date_of_month = date.getDate();
+  var month = date.getMonth()+1;
+  var year = date.getFullYear();
+  var hours = date.getHours();
+  var minutes = date.getMinutes();
+  minutes = minutes < 10 ? '0'+minutes : minutes;
```
Comment (ducnt-0269) [inline app/assets/javascripts/calendar/custom.js]: space +

File: app/assets/javascripts/calendar/custom.js [hunk 7]
```diff
@@ -97,30 +102,32 @@ $(document).ready(function() {
...
-        schedule_start_time.value(start_event);
-        schedule_finish_time.value(finish_event);
+        schedule_start.val(formatDate(start_event));
+        schedule_finish.val(formatDate(finish_event));
       } else if((view.type != "month") && (start._d > date)) {
-        $("#modal-form").modal("show");
-
+        quick_create_schedule.css({
+          display: "block",
+          top:event.pageY - 280,
```
Comment (ducnt-0269) [inline app/assets/javascripts/calendar/custom.js:123]: space top:

File: app/assets/javascripts/calendar/custom.js [hunk 8]
```diff
@@ -210,27 +217,33 @@ $(document).ready(function() {
...
     dayClick: function(date, jsEvent, view) {
       $("#search-setting").hide();
       $("#room_selector, #other_dropdown").removeClass("open");
       if((view.type == "month") && ((date.format() >= (new Date()).toISOString().slice(0, 10)) || (date._d >= (new Date())))) {
-        $("#modal-form").modal("show");
-
-        var TimeZoned = new Date(date.toDate().setTime(date.toDate().getTime() + (date.toDate().getTimezoneOffset() * 60000)));
+        quick_create_schedule.css({
+          display: "block",
+          top:event.pageY - 280,
```
Comment (ducnt-0269) [inline app/assets/javascripts/calendar/custom.js:237]: space top:

File: app/views/pages/home.html.erb [hunk 9]
```diff
@@ -42,10 +42,8 @@
...
 
-<% content_for :modal_form do %>
-  <%= render "shared/modal_form" if current_user %>
-<% end %>
-
 <div class="form-popover hidden"><%= render "popover" %></div>
 
 <div id="time-table" class="hidden"><%= render "time_table" %></div>
+
+<div id="popup" ><%= render "shared/event_short" %></div>
```
Comment (ducnt-0269) [inline app/views/pages/home.html.erb:49]: space "popup" >

**Pull request closed without merging**